### PR TITLE
Fix effects being lost when paused

### PR DESCRIPTION
**Pull request by geekygecko**
## Description

When the app is first open or playback is paused, the player doesn't show that the playback effects are enabled. The reason is that a new PlaybackState is created without the effects. 

I have tried to group the initialisation logic and put it in the PlaybackState object. I'm not sure if it should just in the PlaybackManager. 🤔 

Fixes https://github.com/Automattic/pocket-casts-android/issues/3057

## Testing Instructions

1. Go to Profile tab -> Settings -> General
2. Turn on "Adjust remaining time"
3. Play an episode
4. Open the full screen player
5. Take note of the time remaining
6. Tap on the effects icon
7. Increase the playback speed to 2
8. Close the effect dialog
9. ✅ Verify the time remaining has decreased
10. Pause the episode
11. Move the seekbar to a different position in the episode 
12. ✅ Verify the playback effect icon stays highlight and the remaining time stays decreased
13. Restart the app
14. Open the full screen player
15. ✅ Verify the playback effect icon is highlight and the remaining time is still decreased 

## Video

https://github.com/user-attachments/assets/4638d62d-c280-41a2-ad32-82b0d40fee33

## Checklist
- [ ] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [x] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [x] I have considered whether it makes sense to add tests for my changes
- [x] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [x] Any jetpack compose components I added or changed are covered by compose previews
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.
 
#### I have tested any UI changes...

- [x] with different themes
- [x] with a landscape orientation
- [x] with the device set to have a large display and font size
- [x] for accessibility with TalkBack

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerSeekBar.kt [hunk 1]
```diff
@@ -137,9 +137,9 @@ class PlayerSeekBar @JvmOverloads constructor(context: Context, attrs: Attribute
             val elapsedTime = currentTime.toHhMmSs()
             elapsedTimeText.text = currentTime.toHhMmSs()
             elapsedTimeText.contentDescription = resources.getString(LR.string.player_played_up_to, elapsedTime)
-            val remaingingTime = (-remainingDuration()).toHhMmSs()
-            remainingTimeText.text = remaingingTime
-            remainingTimeText.contentDescription = resources.getString(LR.string.player_time_remaining, remaingingTime.removePrefix("-"))
+            val remainingTime = (-remainingDuration()).toHhMmSs()
+            remainingTimeText.text = remainingTime
+            remainingTimeText.contentDescription = resources.getString(LR.string.player_time_remaining, remainingTime.removePrefix("-"))
```
Comment (geekygecko) [inline modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerSeekBar.kt:142]: Minor typo

File: modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt [hunk 2]
```diff
@@ -2472,18 +2458,17 @@ open class PlaybackManager @Inject constructor(
...
                 state = PlaybackState.State.PAUSED,
-                isSleepTimerRunning = previousPlaybackState?.isSleepTimerRunning ?: false,
-                title = episode.title,
-                durationMs = episode.durationMs,
-                positionMs = episode.playedUpToMs,
-                episodeUuid = episode.uuid,
+                episode = episode,
                 podcast = podcast,
-                chapters = if (sameEpisode) (previousPlaybackState?.chapters ?: Chapters()) else Chapters(),
-                lastChangeFrom = LastChangeFrom.OnUpdatePausedPlaybackState.value,
```
Comment (geekygecko) [inline modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt:2485]: The playback effects were not being added here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,9 @@
         ([#3048](https://github.com/Automattic/pocket-casts-android/pull/3048))
     *   Add support for the Catalan laguage
         ([#3072](https://github.com/Automattic/pocket-casts-android/pull/3072))
+*   Bug Fixes
+    *   Fix effects being lost when paused
+        ([#3084](https://github.com/Automattic/pocket-casts-android/pull/3084))
 *   Updates
```
Comment (mebarbosa) [inline CHANGELOG.md:13]: @geekygecko I've moved this PR to 7.77. Please, make sure you update the changelog of this PR to 7.77 too

**Comment by geekygecko** [comment]
> I tried changing local playback effects and those reverted to the original value in a paused state:

Thanks for finding this, I will work on a fix tomorrow

**Comment by geekygecko** [comment]
I'm going to close this PR for now and reopen it after the work on the Local & Global playback settings project has finished as they are in the same area.

**Pull request closed without merging**